teamC/shinae/app.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config();
+
+const http = require ("http");
+
+const express = require ("express");
+const cors = require ("cors");
+const app = express();
+const morgan = require ("morgan");
+const { DataSource } = require("typeorm");
+
+
+const DataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(()=> {
+      console.log("Data Sourse has been initialized!")
+    })
```

Comment:
```suggestion
const appDataSource = new DataSource({
  type: process.env.TYPEORM_CONNECTION,
  host: process.env.TYPEORM_HOST,
  port: process.env.TYPEORM_PORT,
  username: process.env.TYPEORM_USERNAME,
  password: process.env.TYPEORM_PASSWORD,
  database: process.env.TYPEORM_DATABASE
})

appDataSource.initialize()
    .then(()=> {
      console.log("Data Sourse has been initialized!")
    })
```
신애님이 현재 작성해주신 코드로는 nodemon app.js와 같은 커맨드로 불러왔을 때 정상적으로 동작하지 않을 것 같네요! 

추가로 appDataSource.initialize() 부분에서 .then에 이어 .catch(err) {} 코드도 위에 코멘트에 남겨드린 공식문서를 참고하여 체이닝해주세요! 

어려우신게 있으시면 Q&A 방으로 와주세요 :)